tidb_exporter/README.md

```diff
@@ -1,7 +1,7 @@
 TiDB exporter
 ------
 
-TiDB exporter for Prometheus. The `tidb_exporter` will expose metrics via HTTP on the `/metrics` endpoint. Currently only one tidb server address is supported. It is recommended to set load balancing IP. We'll support multiple tidb servers in the near future.
+TiDB exporter for Prometheus. Checking tidb server's health by sending SQL query to it. The `tidb_exporter` will expose metrics via HTTP on the `/metrics` endpoint. 
```

Comment:
Change this part to:

TiDB exporter (for Prometheus) is a tool for checking the TiDB server's health by sending SQL queries to it. The `tidb_exporter` exposes metrics via HTTP on the `/metrics` endpoint.